src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonExtension.cs

```diff
@@ -49,11 +50,12 @@ public void Setup(MarkdownPipeline pipeline, IMarkdownRenderer renderer)
         }
     }
 
-    public interface ITripleColonExtensionInfo
-    {
-        string Name { get; }
+	public interface ITripleColonExtensionInfo
+	{
+		string Name { get; }
+		bool SelfClosing { get; }
         bool TryProcessAttributes(IDictionary<string, string> attributes, out HtmlAttributes htmlAttributes, Action<string> logError);
         bool TryValidateAncestry(ContainerBlock container, Action<string> logError);
-        // todo: "Render" function as-needed.
+		Func<HtmlRenderer, TripleColonBlock, bool> Render { get; }
```

Comment:
Use `bool Render(HtmlRenderer, TripleColonBlock)` over delegate for this interface

Reply:
Added.